### ISSUE TITLE
boot.rom Issues

### ISSUE THREAD

**Issue by bebopjp** [MiSTer-devel/Amstrad_MiSTer#27]
The latest boot.rom wouldn't work properly.
I booted set to the 6128 but it said it had 64K RAM.
I also expected it to boot Basic 1.1, but booted Basic 1.0.
It just kept saying "press PLAY then any key" when I ran CAT.
The workaround was to download the amstrad.rom from this repo: https://github.com/sorgelig/Amstrad_MiST/tree/master/releases
Then change the name to boot.rom.  Everything worked great after that (reported 128K RAM and Basic 1.1 as well)

**Comment by sorgelig** [comment]
it haven't been changed for years. Something specific to your setup may be. It works fine for me.

**Comment by bebopjp** [comment]
Ah, I see what I did now.

To recreate, I first formatted a fresh SD, ran update_all, and saw the issue went away.

Then, I tried messing with the expansion ROM, cpc464nd.ez40

After I applied that, it was in this state permanently.  Tried reloading the core, power cycling the Mister, but it would always load the expansion ROM after this.  Is there a way to un-apply the expansion ROM with the OSD, or will I have to mess with the config file to undo the 464 mode?

**Comment by bebopjp** [comment]
Ok, so I just deleted everything Amstrad* from the config folder and that undid the expansion ROM modification.

Not to muddy the waters (since I'm still very new to BASIC and the CPC), but here's how I got into this mess in the first place.

I was trying to get "All Points Bulletin" to load.  When running CAT, I saw two files (note the * at the end of the filename):

apb.bin*
disc.bin*

I couldn't get either to run properly, just got a empty blue screen with every core config, so I tried applying the expansion ROM assuming the APB rom was incompatible with the 6128 or the 664.

I've tried many games since then with no issue, the only thing weird thing I noticed that was different about APB was the bin files have a * at the end when running CAT.  Never seen that before and don't know what it means.
